### ISSUE TITLE
add synthetic tags to parts of the result

### ISSUE THREAD

**Issue by jongleur1983** [drolbr/Overpass-API#49]
Whil I'm Implementing a table view for the overpass-ide (overpass-turbo) someone suggested not to show elements without tags, as these would not be interesting for the user.
IMHO it's slightly different: Objects returned by overpass (only) due to recurse statements are likely to be not of interest.
But it's not possible to identify these objects without reworking the query again.

Therefore a QL command to add custom tags to a data set would be great.
It would e.g. allow to
- select stuff
- add the tag "_self_selected"="true"
- recurse down

stuff could contain nodes and ways.
the result would contain nodes from stuff as well as nodes from the recursion, but only nodes from stuff would contain the _self_selected-Tag.

**Comment by mmd-osm** [comment]
@jongleur1983 : by using `[~"."~"."]` you can make sure that any element you query has at least one tag. Also `out geom` could be interesting:

Here's an example for ways:

```C
[bbox:{{bbox}}];
way[highway];
out center tags;
>;
node._[~"."~"."];
out geom tags;
```

overpass turbo link: http://overpass-turbo.eu/s/754

**Comment by drolbr** [comment]
Party solved by the geometry modes. Other parts solved by the convert statement since 0.7.54.
